Octokit.Reactive/Clients/IObservableIssuesEventsClient.cs

```diff
@@ -14,8 +20,17 @@ public interface IObservableIssuesEventsClient
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="number">The issue number</param>
-        /// <returns></returns>
         IObservable<EventInfo> GetAllForIssue(string owner, string name, int number);
+        
+        /// <summary>
+        /// Gets all events for the issue.
+        /// </summary>
+        /// <remarks>
+        /// http://developer.github.com/v3/issues/events/#list-events-for-an-issue
+        /// </remarks>
+        /// <param name="repositoryId">The ID of the repository</param>
```

Comment:
This one is another `ID` versus `id` thing. I'm not sure which one is better (or more correct).

@ryangribble @haacked feels?